patches/DP-30894_views_bulk_edit_remove_paragraph_fields_from_bulk_form.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/src/Form/BulkEditFormTrait.php b/src/Form/BulkEditFormTrait.php
+index de8ef20..12105bd 100644
+--- a/src/Form/BulkEditFormTrait.php
++++ b/src/Form/BulkEditFormTrait.php
+@@ -255,6 +255,10 @@ trait BulkEditFormTrait {
+     $definitions = $this->entityFieldManager->getFieldDefinitions($entity_type_id, $bundle);
+ 
+     foreach (Element::children($form) as $key) {
++      if (isset($form[$key]['widget'][0]['#paragraph_type'])) {
++        unset($form[$key]);
++        continue;
++      }
+       if (isset($form[$key]['#access']) && !$form[$key]['#access']) {
```

Comment:
This doesn't look typical to me. usually #access is evaluated by render API and items are not shows there. It seems like we are doing the renderer's job here.

Reply:
Good to know that this is temporary code, but it still may last in our codebase for years. It doesn't pass [peer review guidelines](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md) IMO, temporary or not. If anyone else thinks differently, then they can record an internal review approval. I suggest  adding code comments indicating why we are bypassing the render system and when the bypass can be removed.